### ISSUE TITLE
In "typescript-compiler" package, when registering modules, remove the `.ts` file extension

### ISSUE THREAD

**Issue by Urigo** [Urigo/angular2-meteor#80]
Should be related to how we define the path in the compiler, for example:
https://github.com/barbatus/ts-compilers/blob/master/compilers/ts_caching_compiler.js#L104

* I don't think we need issue to be resolve in order to do that but I'm not sure - https://github.com/meteor/meteor/issues/6034

**Comment by dotansimha** [comment]
@barbatus 
So now in order to load a local TypeScript file in need to do:
```js
import {MyClass} from '../collections/data.ts';
```

instead of just use:
```js
import {MyClass} from '../collections/data';
```

or:
```js
import {MyClass} from 'collections/data';
```

I think that you can declare the module's name in the compiler

**Comment by barbatus** [comment]
Has anything changes since beta-4?  Last time I looked into this Meteor 1.3  was to blame since it always registered module with an extension. See the above mentioned issue.

**Comment by dotansimha** [comment]
@barbatus 
Nothing changed in `beta.5` of Meteor 1.3, just checked... Are you sure that this is not related to the `ts-compilers` package? maybe here https://github.com/barbatus/ts-compilers/blob/master/compilers/ts_caching_compiler.js#L112

**Comment by barbatus** [comment]
did you read the issue https://github.com/meteor/meteor/issues/6034?

**Comment by dotansimha** [comment]
@barbatus Oh sorry, did not noticed it

**Comment by barbatus** [comment]
Already fixed some time ago in Meteor 1.3 RC.
